docs/architecture/sdk/server-bindings.md

```diff
@@ -22,12 +22,14 @@ let response: SyncResponseModel =
     bitwarden_api_api::apis::sync_api::sync_get(&config.api, exclude_subdomains).await?;
 ```
 
-It's highly recommended to not expose the request and response models of the generated bindings and
-instead use your own models. This decouples the server request / response models from the SDK
-models, and allows for easier changes in the future.
+You _should not_ expose the request and response models of the auto generated bindings and _should_
+instead define and use your own models. This ensures the server request / response models are
```

Comment:
question: How do we refer to these models? domain models?

Reply:
:shrug:, everything's a domain model in the end. Just different domains 😭

Reply:
@Hinton ah fair enough, we'll figure it out as we go